### ISSUE TITLE
TruffleRuby linker doesn't support `-I` option without space

### ISSUE THREAD

**Issue by fniephaus** [oracle/truffleruby#1708]
TruffleRuby linker doesn't support `-I` (capital i) option without space

### Steps to Reproduce

1. Install for example tree-sitter gem (`gem install tree-sitter`)
  * In order to do so, `TREE_SITTER_PARSER_DIR` has to be set to a directory of parser files, [for example these files](https://github.com/tree-sitter/ruby-tree-sitter/tree/master/test/tree-sitter/fixtures/parsers/python)

### Expected Behavior

Installs without error

### Actual Behavior

```
linking shared-object tree-sitter/treesitter.su
/path/to/graalvm-ce-19.0.0/Contents/Home/jre/languages/ruby/lib/cext/linker.rb:67:in `process_args': Unknown argument: -I/path/to/graalvm-ce-19.0.0/Contents/Home/jre/languages/ruby/lib/ruby/gems/2.6.0/gems/tree-sitter-0.1.0/ext/tree-sitter/tree-sitter/include (RuntimeError)
        from /path/to/graalvm-ce-19.0.0/Contents/Home/jre/languages/ruby/lib/cext/linker.rb:21:in `main'
        from /path/to/graalvm-ce-19.0.0/Contents/Home/jre/languages/ruby/lib/cext/linker.rb:150:in `<main>'
make: *** [treesitter.su] Error 1
```

### Workaround

add this case

```ruby
when /\A-I.+\z/
   @incflags << arg

```

to the arg parsing case in [`/path/to/graalvm-ce-19.0.0/Contents/Home/jre/languages/ruby/lib/cext/linker.rb` (for example line 50)](https://github.com/oracle/truffleruby/blob/master/lib/cext/linker.rb#L50)

*Tested with GraalVM CE 19.0.0 on macOS 10.14.5.*

/cc @querenker

**Comment by eregon** [comment]
This should be fixed in 19.3.0 because we use a native linker now.
@fniephaus Please confirm. I'll close this issue as the report bug should be fixed.

I could not reproduce because it seems both `TREE_SITTER_PARSER_DIR` and emscripten need to be installed for installing this gem.
BTW it takes a long time to install this gem, probably because it includes 2069 files and they are extracted one by one.

**Comment by fniephaus** [comment]
Thanks, I will try again when `19.3.0` is out.

**Comment by eregon** [comment]
@fniephaus TruffleRuby 19.3.0 was released, could you verify it works now?

**Comment by fniephaus** [comment]
This issue seems to be fixed with `19.3.0`. Thanks!
